### PR TITLE
Report

### PR DESCRIPTION
**Pull request by sudo-joseph**

### CODE REVIEW

File: apps/core/views.py [hunk 1]
```diff
@@ -1,48 +1,71 @@
...
+                       'vlon': request.session['lon'],
+                       'view': '18',
+                       }
+            return render(request, 'pages/add_report.html', context)
+
+def log_location(request):
+    loc = json.loads(request.body)
+    request.session['lat'] = loc['lat']
+    request.session['lon'] = loc['lon']
+    return HttpResponse(request)
```
Comment (ishinan) [inline apps/core/views.py:68]: What does `HttpResponse(request)` do?

**Review by ishinan — APPROVED** [review]
one comment was added. 
Great! approved!

File: apps/core/views.py (hunk 1 above)
Comment (sudo-joseph) [reply apps/core/views.py:68]: Sends a response to the request here. Without it, django complains.

Comment (sudo-joseph) [reply apps/core/views.py:68]: Its not doing anything though, just acknowledging that the we got the the POST request.

**Pull request merged**